src/modules/staking/services/staking.compute.service.ts

```diff
@@ -991,4 +992,74 @@ export class StakingComputeService {
 
         return true;
     }
+
+    async getAllBaseAndMaxBoostedAPRs(
```

Comment:
I would use the existing compute methods for base and boosted APR: `computeBoostedAPR` `computeMaxBoostedApr`